web/onload.js

```diff
@@ -15,7 +15,8 @@ $(function() {
   if (the.use_codemirror && typeof CodeMirror !== 'undefined') {
 
     the.editor = CodeMirror.fromTextArea(textArea, {
-      lineNumbers: true
+      lineNumbers: true,
+      lineWrapping: true,
```

Comment:
We don't want to turn on codemirror line wrapping as it might be different from the wrapping the beautifier does.